test/hotspot/jtreg/vmTestbase/nsk/jdb/kill/kill001/kill001.java

```diff
@@ -50,12 +50,13 @@
  * @run main/othervm
  *      nsk.jdb.kill.kill001.kill001
  *      -arch=${os.family}-${os.simpleArch}
- *      -waittime=5
+ *      -waittime=1 -verbose
```

Comment:
please add -verbose at separate line.
And I suppose you changed waittime temporary and forgot to revert it.
The same for other tests

Reply:
Ok, will move to separate line. For `-waittime`, 5 seems overkill for all these tests. If anything ever goes wrong with the test, it seems you always end up waiting for a timeout. So first thing I do is change `-waittime` to 1 so I don't have to wait as long when debugging the test. I'd change it to .5 if I could, but it doesn't accept decimals. I could change it back to 5, but I don't see why it needs to be so big. Maybe @lmesnik can comment on this.